library/ts-rollup/src/utils.ts

```diff
@@ -0,0 +1,6 @@
+// naive implementation, doesn't prevent duplicates in grid
+export function getRandomInteger(): number {
+  const min = 1;
+  const max = 100;
```

Comment:
⚠️ I suggest we update the README bingo rules as it currently states that "Bingo is a game of chance played with 90 balls, numbered from 1 to 90,"
Updating the readme avoids to change the screenshots